docs/guides/install/index.html

```diff
@@ -3,215 +3,248 @@
 ---
 {% extends 'guide.njk' %}
 {% block content %}
-<section>
-  <h2 class="hxSectionTitle" id="assets">Assets</h2>
-  <table class="hxTable hxTable--condensed">
-    <thead>
-      <tr>
-        <th></th>
-        <th>Development (unminified)</th>
-        <th>Production (minified)</th>
-      </tr>
-    </thead>
-    <tbody>
-      <tr>
-        <th>Appearance</th>
-        <td>
-          <a href="{{data.url.cdn}}/{{data.cdn.css.helixUi}}" target="_blank">
-            <hx-icon type="external-link"></hx-icon>
-            helix-ui.css
-          </a>
-        </td>
-        <td>N/A</td>
-      </tr>
-      <tr>
-        <th>Behavior</th>
-        <td>
-          <a href="{{data.url.cdn}}/{{data.cdn.js.helixUi}}" target="_blank">
-            <hx-icon type="external-link"></hx-icon>
-            helix-ui.js
-          </a>
-        </td>
-        <td>N/A</td>
-      </tr>
-    </tbody>
-  </table>
-</section>
-
-<section><!-- Web Components -->
-  <h2 class="hxSectionTitle" id="web-components">Web Components</h2>
-  <section>
-    <table class="hxTable hxTable--condensed">
-      <thead>
-        <tr>
-          <th></th>
-          <th>Production (CDN)</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <th>ES5 Adapter</th>
-          <td>
-            <a href="{{data.url.cdn}}/{{data.cdn.js.es5adapter}}" target="_blank">
-              <hx-icon type="external-link"></hx-icon>
-              custom-elements-es5-adapter.js
-            </a>
-          </td>
-        </tr>
-        <tr>
-          <th>Web Components Loader</th>
-          <td>
-            <a href="{{data.url.cdn}}/{{data.cdn.js.polyfillLoader}}" target="_blank">
-              <hx-icon type="external-link"></hx-icon>
-              webcomponents-loader.js
-            </a>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-
-    <p>
-      To begin using the custom elements defined in HelixUI, you must first
-      ensure that the browser can load them. To do so, you'll need to insert
-      both the ES5 Adapter and the Polyfill Loader into your markup.
-    </p>
+  <section><!-- Install from NPM -->
+    <h2 class="hxSectionTitle" id="install">Install from NPM</h2>
```

Comment:
You can choose to ignore this comment, or address it later :). I would be interested to know what NPM is, and why I need to install from NPM. Like add a bit more context in the intro. Also, since NPM is an acronym, upon first use you should spell out the full name, and then add (NPM) in parentheses. All other times you can simply use 'NPM'.

Reply:
This is a good point. I'll use jQuery's [install page](http://jquery.com/download/#downloading-jquery-using-npm-or-yarn) as inspiration to update this copy.